lang/java/compiler/src/main/java/org/apache/avro/compiler/specific/SpecificCompiler.java

```diff
@@ -225,6 +268,14 @@ public void setEnableDecimalLogicalType(boolean enableDecimalLogicalType) {
     this.enableDecimalLogicalType = enableDecimalLogicalType;
   }
 
+  public boolean useJodaForDateTimeLogicalTypes() {
```

Comment:
any reason not to expose the enum here? the two getters are not independent, and the interface essentially exposes four possible states when there are only two.